site/pages/10--docs/20--basics/20--pages.rocket.md

```diff
@@ -189,7 +189,7 @@ export { html, layout };
 
 Rocket pages can fetch data to help generate your pages. Rocket uses two platform features to get this out of the box: **fetch()** and **top-level await.**
 
-📚 Read our [full guide](/en/guides/data-fetching) on data-fetching to learn more.
+📚 Read our [full guide](../30--guides/80--data-fetching.rocket.md) on data-fetching to learn more.
```

Comment:
I noticed a very strange bug here @daKmoR . for some reason this URL gets transformed to `http://localhost:8000/docs/guides/data-fetching/d%22`. no idea where the extra `d"` comes from. I can only assume that it's as a result of some bad markdown formatting 🤷🏽

Reply:
It's curious because when I log out the result of `sourceRelativeFilePathToUrl(path.join(dir, url));` inside of `AdjustAssetUrls.js`, the result is `/docs/basics/data-cascade/`. which is good!

Reply:
Ha! after doing some digging, it looks like it has something to do with the how the Markdown parser treats paragraphs that begin with emojis...

with emoji:

<img width="372" alt="image" src="https://user-images.githubusercontent.com/17087167/160252266-19d5320e-9140-4e9f-85e3-c5bc980e257f.png">


without emoji:

<img width="331" alt="image" src="https://user-images.githubusercontent.com/17087167/160252273-042273a3-5065-4376-aec4-8b4f140b9229.png">

Reply:
After doing a little more investigating, it seems that the issue occurs only when the emoji is infront of the URL. If it's placed afterward, then the URL is generated perfectly fine... 🤔

Reply:
now that you mention it... I remembered that on rocket main we had this as well... 
utf8 encoding and decoding was the solution if I remember correctly...

https://github.com/modernweb-dev/rocket/blob/14721d1e0f8621089ee68420e4484df671f2b7f5/packages/cli/src/eleventy-plugins/processLocalReferences.cjs#L209

I really hope there is a simpler solution then to add a dependency for it